.github/renovate.json

```diff
@@ -1,6 +1,44 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "local>axonivy/renovate-config"
+    "local>axonivy/renovate-config",
+    "schedule:earlyMondays"
+  ],
+  "baseBranches": [
+    "master",
+    "release/12.0",
+    "release/10.0"
+  ],
```

Comment:
why needed. should be defined in the base config

Reply:
one place more to maintain stuff if we create a new release branch
Isn't it possible to do a release/* branches thing?

Reply:
I've failed in the past to setup a pattern, but it seem neatly document, so let's give a try :crossed_fingers: 
https://docs.renovatebot.com/configuration-options/#basebranches

at least my config is valid ... a real run will be required to see whether it works:
```
Now using node v22.14.0 (npm v10.9.2)
❯ renovate-config-validator
 INFO: Validating .github/renovate.json
 INFO: Config validated successfully
 ~/de/r/web-tester  automerge.maven         
```

Reply:
seem like we can't inherit `baseBranches` :shrug: 

https://github.com/axonivy/web-tester/pull/392